### PR TITLE
Added tool Fasttree

### PR DESCRIPTION
**Pull request by matthew-fogel**
Adding Fasttree galaxy wrapper to tools-iuc

### CODE REVIEW

File: tools/fasttree/fasttree.xml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+                <param name="likelihood" argument="-noml" type="boolean" truevalue="-noml" falsevalue="" checked="false" label = "Turn off maximum-likelihood." />
+                <param name="fastest" argument="-fastest" type="boolean" truevalue="-fastest" falsevalue="" checked="false" label="Speed up the neighbor joining phase and reduce memory usage (recommended for >50,000 sequences)" />
+                <param name="pseudo" argument="-pseudo" type="boolean" truevalue="-pseudo" falsevalue="" checked="false" label="Use pseudocounts" help="Recommended for highly gapped/fragmentary sequences." />
+                <param name="mllen" argument="-mllen" type="boolean" truevalue="-mllen" falsevalue="" checked="false" label="Optimize branch lengths without ML NNIs." />
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="output" format="nhx" label="${tool.name} on ${on_string}:tree.nhx" />
+        <data name="log" format="txt" hidden="true" label="${tool.name} on ${on_string}:log_FastTree_run.txt" />
```
Comment (bgruening) [inline tools/fasttree/fasttree.xml]: Should this log file be an optional one?

File: tools/fasttree/fasttree.xml [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+=========
+Resources
+=========
+
+FastTree_v2.1.10_
+
+.. _FastTree_v2.1.10: http://meta.microbesonline.org/fasttree/#Install
+]]>
+    </help>
+    <citations>
```
Comment (bgruening) [inline tools/fasttree/fasttree.xml]: We should cite this tool with: doi:10.1371/journal.pone.0009490

File: tools/fasttree/fasttree.xml (hunk 1 above)
Comment (matthew-fogel) [reply tools/fasttree/fasttree.xml]: Yes, that probably would be better. I have made it optional now.

File: tools/fasttree/fasttree.xml (hunk 2 above)
Comment (matthew-fogel) [reply tools/fasttree/fasttree.xml]: Okay, citation added.

**Comment by bgruening** [comment]
Super cool, thanks a bunch!

**Pull request merged**